lib/siwe/message.rb

```diff
@@ -76,47 +77,41 @@ class Message
 
     def initialize(domain, address, uri, version, options = {})
       @domain = domain
-      begin
-        @address = Eth::Address.new(address).to_s
-      rescue StandardError
-        raise Siwe::InvalidAddress
-      end
-      raise Siwe::InvalidAddress unless @address.eql? address
-
+      @address = address
       @uri = uri
       @version = version
       @statement = options.fetch :statement, ""
       @issued_at = options.fetch :issued_at, Time.now.utc.iso8601
       @nonce = options.fetch :nonce, Siwe::Util.generate_nonce
-      @chain_id = options.fetch :chain_id, "1"
+      @chain_id = options.fetch :chain_id, 1
       @expiration_time = options.fetch :expiration_time, ""
       @not_before = options.fetch :not_before, ""
       @request_id = options.fetch :request_id, ""
       @resources = options.fetch :resources, []
+      validate
```

Comment:
this would raise an error if the SiweMessage is expired, right? imo, that should not be the case.

Reply:
No, validate only checks for the grammar, verify would though